### DISCUSSION TITLE
Momentary push button into switch on/off

### DISCUSSION THREAD

**Discussion by xoanxil** [tttapa/Control-Surface#830] [Q&A]
Hi,

I made a search but I didn´t find information abouit how to make a momentary push button working as a switch. Is this possible using code?

CCButton button[] {
  {0, {MIDI_CC::Sound_Controller_1} },
  {1, {MIDI_CC::Sound_Controller_2} },
  {2, {MIDI_CC::Sound_Controller_3} },
  {3, {MIDI_CC::Sound_Controller_4} },
  {5, {MIDI_CC::Sound_Controller_5} },
  {7, {MIDI_CC::Sound_Controller_6} },
  {11, {MIDI_CC::Sound_Controller_7} },
  {13, {MIDI_CC::Sound_Controller_8} },
};

Thanks

**Comment by tttapa** [comment]
Have a look at <https://tttapa.github.io/Control-Surface-doc/Doxygen/d7/dcd/group__MIDIOutputElements.html>.

You probably want the `CCButtonLatched` class.

**Comment by xoanxil** [comment]
Thanks a lot!

Best
